mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -1047,6 +1073,132 @@ impl TendermintCoin {
         Box::new(fut.boxed().compat())
     }
 
+    pub(super) async fn get_sender_trade_fee_for_denom(
+        &self,
+        ticker: String,
+        denom: Denom,
+        decimals: u8,
+        amount: BigDecimal,
+    ) -> TradePreimageResult<TradeFee> {
+        const TIME_LOCK: u64 = 1750;
+        let sec: [u8; 32] = thread_rng().gen();
```

Comment:
Can we use a dummy secret and hash? For example, `[0; 32]` and `H256::default()`

Reply:
Secrets are used in calculating `hash_lock` and htlc ids. Sending same `secret` might raise errors on transactions. We create random secrets for tests because of the same reason.

Timestamp(one of the other value used for hash_lock and htlc id calculations) is set to `0` for all transactions in mm2, if we would send current timestamp for tendermit transactions, then we could do what you said. But sometime ago, Artem said that we don't use timestamp for other htlcs so it's better to not send it for tendermint either.

Reply:
Got it, thank you!